### PR TITLE
Write introduction

### PR DESCRIPTION
**Pull request by daffidwilde**
Introduction chapter consisting of:

- [x] Summary of motivation
- [x] Thesis structure
- [x] ~~Research questions~~ Novel contributions
- [x] Software and best practices
- [x] Figure showing connections between chapters

### CODE REVIEW

File: chapters/intro/main.tex [hunk 1]
```diff
@@ -1,26 +1,165 @@
...
+system. The journey to this point has not been without its deviations from this
+primary task, but that objective has been achieved.
+
+With an administrative dataset provided by \ctmuhb, exploratory analysis found
+that the population in question was deeply varied and heterogeneous --- as
+expected. However, the dataset was insufficiently detailed to construct
+meaningful models on the entire population or system, opening up an interest in
+population segmentation techniques. While data requests were made, the gathering
+of literature began. Even with the expansive nature of machine learning
+literature, two clear patterns emerged.
```
Comment (drvinceknight) [inline chapters/intro/main.tex:18]: This is good but perhaps could be worded in a different way. Perhaps a more generic: "common problems in applying machine learning to healthcare are ... the contribution of this thesis is to use population segmentation ..."

File: chapters/intro/main.tex [hunk 2]
```diff
@@ -1,26 +1,165 @@
...
+conclusions drawn from that process. Often, a method would be deemed
+`state-of-the-art' or `better' based on a process that offered little insight
+into the actual quality of that method. This reliance on a narrow assessment
+process prompted research into how else an algorithm could be evaluated
+objectively.
+
+Over the course of this project, it became clear that the requests for more
+detailed datasets would not pan out in time, and so the task became one of
+extracting as much actionable information from the data at hand. Simultaneously,
+\ctmuhb\ provided further administrative datasets which related to the members
```
Comment (drvinceknight) [inline chapters/intro/main.tex:51]: I think the style of these first 3 paragraphs is not right, things like "would not pan out in time" feel more reflective (and could perhaps be in your conclusion) but they should not be in your introduction. The introduction should be more upbeat and to the point:

- What's the problem?
- What's the solution approach?
- How are you going to write it out?

File: chapters/intro/main.tex [hunk 3]
```diff
@@ -32,22 +171,220 @@ \subsection{Code snippets}
...
+reinforcing the longevity of the software. Table~\ref{tab:repos} details the
+repositories and archives associated with each chapter, where appropriate.
+
+\begin{table}[tbhp]
+    \centering%
+    \resizebox{\textwidth}{!}{%
+        \input{chapters/intro/tex/repos/main.tex}%
+    }\caption{%
+        A summary of the repositories and archives associated with the chapters
+        of this thesis%
```
Comment (drvinceknight) [inline chapters/intro/main.tex:382]: I wonder if you could add a sentence (or brackets) explaining when it's not appropriate (`due to patient data confidentiality`) or something like that?

**Review by drvinceknight — CHANGES_REQUESTED** [review]

File: chapters/intro/main.tex (hunk 3 above)
Comment (daffidwilde) [reply chapters/intro/main.tex]: Yeah, I had thought that myself. Added a sentence in 8b77263.

**Comment by daffidwilde** [comment]
@drvinceknight the opening is ready for your review. I think it's pretty succinct (in a good way) but let me know what you think.

I'm stuck on how to approach writing the "research questions and novel aspects" section like we discussed last week. I agree that it should be in there, but I don't know how to rephrase the questions. I'll drop a line to Jon asking for advice on that as well.

File: chapters/intro/main.tex [hunk 4]
```diff
@@ -1,26 +1,167 @@
...
+treating and living with COPD inherently difficult, emphasising the importance
+of studying it closely.
+
+This thesis incorporates three seemingly disparate themes related to modern OR:
+the evaluation of algorithms, clustering and segmentation, and operational
+healthcare modelling. The chapters of this thesis provide novel methods for
+evaluating an algorithm's performance and incorporating game theory into an
+existing clustering algorithm. In turn, these methods contribute to a novel
+operational methodology, which addresses the concerns of \ctmuhb\ and their COPD
+population, despite a lack of highly detailed data.
```
Comment (drvinceknight) [inline chapters/intro/main.tex:54]: Great paragraph.

**Pull request merged**